daisy_workflows/build-publish/windows/windows-81-ent-x64-uefi.publish.json

```diff
@@ -0,0 +1,45 @@
+{{/*
+  Template to publish Windows server UEFI images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-81-ent-x64-uefi",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{if eq .environment "staging" -}}
+  "WorkProject": "bct-staging-images",
+  "PublishProject": "bct-staging-images",
+  "ComputeEndpoint": "https://www.googleapis.com/compute/staging_alpha/projects/",
+  "DeleteAfter": {{$delete_after}},
+  {{- else -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "google.com:windows-internal",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- end}}
+  {{$guest_features := `["MULTI_IP_SUBNET","UEFI_COMPATIBLE","VIRTIO_SCSI_MULTIQUEUE","GVNIC","WINDOWS"]` -}}
+  {{$time := trimPrefix .publish_version "v"}}
+  "Images": [
+    {
+      "Prefix": "windows-81-ent-x64",
+      "Family": "windows-81",
+      "Description": "Microsoft, Windows 8.1 Enterprise, x64 built on {{$time}}, supports Shielded VM features",
+      "Architecture": "X86_64",
+      "Licenses": [
+        {{if eq .environment "staging" -}}
+        "projects/windows-cloud/global/licenses/windows-81-x64-byol"
```

Comment:
This says Windows 8.1 uses `windows-8-x64-byol` as the license.

https://cloud.google.com/compute/docs/images/os-details#windows_client

Also, should the family and prefix just be `windows-8`?

Reply:
As for the family, I think we want distinction between 8 and 8.1 (there are people still using 8, somehow). Admittedly I did just choose the same prefix as I had seen in the internal versions of the files I'm porting over to github. I suspect this would keep it consistent with previous iterations.

Reply:
8 and 8.1 are actually different Windows versions (despite the fact that they appear to be point releases). They have different kernel behaviors. Also, these images are largely for internal testing.